src/QmlControls/CameraWarningBox.qml

```diff
@@ -0,0 +1,136 @@
+﻿import QtQuick          2.12
+import QtQuick.Controls 2.4
+import QtQuick.Layouts  1.11
+import QtQuick.Dialogs  1.3
+
+import QGroundControl                       1.0
+import QGroundControl.Controls              1.0
+import QGroundControl.Palette               1.0
+import QGroundControl.MultiVehicleManager   1.0
+import QGroundControl.ScreenTools           1.0
+import QGroundControl.Controllers           1.0
+import QtGraphicalEffects                   1.12
+
+Rectangle {
+    id: root
+    color: "#222020"
+    z: 3
+
+    radius: 12
+    property var rootWidth: null;
+    property var rootHeight: null;
```

Comment:
The rootWidth and rootHeight properties are declared but not bound to any values. Consider binding these properties to the actual screen dimensions or relevant parent dimensions.

property int rootWidth: Screen.width
property int rootHeight: Screen.height